lms/static/js/edxnotes/plugins/accessibility.js

```diff
@@ -0,0 +1,234 @@
+;(function (define, undefined) {
+'use strict';
+define(['jquery', 'underscore', 'annotator'], function ($, _, Annotator) {
+    /**
+     * Adds the Accessibility Plugin
+     **/
+    Annotator.Plugin.Accessibility = function () {
+        // Call the Annotator.Plugin constructor this sets up the element and
+        // options properties.
+        Annotator.Plugin.apply(this, arguments);
+    };
+
+    $.extend(Annotator.Plugin.Accessibility.prototype, new Annotator.Plugin(), {
+        pluginInit: function () {
+            this.annotator.subscribe('annotationViewerTextField', _.bind(this.addAriaAttributes, this));
+            this.annotator.element.on('keydown', '.annotator-hl', _.bind(this.onHighlightKeyDown, this));
+            this.annotator.element.on('keydown', '.annotator-viewer', _.bind(this.onViewerKeyDown, this));
+            this.annotator.element.on('keydown', '.annotator-editor', _.bind(this.onEditorKeyDown, this));
+            this.addTabIndex();
+        },
+
+        destroy: function () {
+            this.annotator.unsubscribe('annotationViewerTextField', this.addAriaAttributes);
+            this.annotator.element.off('keydown', '.annotator-hl');
```

Comment:
@polesye 
> We should unbind only a11y plugin related callbacks.

You are correct, done -- I've namespaced the events we are binding in this plugin:
`keydown.accessibility.hl`
`keydown.accessibility.viewer`
`keydown.accessibility.editor`